### ISSUE TITLE
File Associate for .sqlite 

### ISSUE THREAD

**Issue by xmha97** [sqlitebrowser/sqlitebrowser#2729]
Please associate `.sqlite` file.
![image](https://user-images.githubusercontent.com/23485114/120080833-d3da6280-c0cf-11eb-95d3-f5b46b16d40f.png)

**Comment by chrisjlocke** [comment]
Are you using a Windows operating system?  Have you installed DB4S, or just extracted the .zip file and run the .exe ?

![image](https://user-images.githubusercontent.com/7872218/120082860-3b3ae700-c0bd-11eb-9810-497ddbd95710.png)

You can right-click an .sqlite file, and click 'Change'.

**Comment by xmha97** [comment]
> DB4S

Yes, I have now manually added the app to the "Open With" menu.
But I want the installer to do this automatically. (Like all other apps)

**Comment by justinclift** [comment]
Yeah, agreed.  It's something we'll add for a future installed.  We've done some initial investigation, and it shouldn't be **_tooooo hard_**. :smile:
